lib/index.js

```diff
@@ -150,6 +150,10 @@ function MqttProtocol () {
       }
     }
   };
+
+  this.disconnect = function (clientId) {
```

Comment:
Quickly checking mosca documentation, I think the `onConnection`method should also store the provided `client`object.
This allows the `disconnect` method to call `client.close([callback], [reason])`, actively closing the client's connection.